recipes/ed25519-blake2b/1_rename-edsig-edsig_blake2b.patch

```diff
@@ -0,0 +1,188 @@
+--- bin/edsig      2024-02-16 16:57:39.437391589 -0600
```

Comment:
Is the diff smaller if you combine patches 1 and 2? (the rename and the replacement of edsig with edsig_blake2b)?

Reply:
I tried but it did not work, it complained that I was deleting a file that did not exist, but I could have mistyped something. That was my preferred route s well, let me take a look again

Reply:
```
Applying patch: /home/conda/staged-recipes-copy/recipes/ed25519-blake2b/2_fix-edsig_blake2b.patch
Applying patch: /home/conda/staged-recipes-copy/recipes/ed25519-blake2b/2_fix-edsig_blake2b.patch with args:
['-Np2', '-i', '/tmp/tmppivo3ljo/2_fix-edsig_blake2b.patch.native', '--binary']
checking file bin/edsig
patching file bin/edsig
```
then
`error: [Errno 2] No such file or directory: 'bin/edsig_blake2b'`

Oh, maybe the rename should be empty?

Reply:
Ok, got it, you're point is that since I have a big patch anyway, I should skip `2_...`. I thought it'd be clearer to verify that `1_` does not change anything but the file name, then `2_...` shows the changes